src/services/query-options.js

```diff
@@ -31,6 +31,15 @@ class QueryOptions {
       options.limit = this._limit;
     }
 
+    if (this._restrictFieldsOnRootModel && this._requestedFields.size) {
+      // Restricting loaded fields on the root model is optin with sequelize to avoid
```

Comment:
```suggestion
      // Restricting loaded fields on the root model is an option with sequelize to avoid
```

Reply:
I meant that that it is an opt-in behavior
As the opposite of opt-out

I added the missing dash